client/apps/plugin-status/health.js

```diff
@@ -25,8 +27,32 @@ const HealthView = ( { translate, healthItems } ) => {
 			<Indicator
 				title={translate('Jetpack')}
 				state={healthItems.jetpack.state}
-				message={healthItems.jetpack.message}/>
-			<WooCommerceServicesIndicator />
+				message={healthItems.jetpack.message}
+			/>
+			<Indicator
+				title={translate('Automatic Taxes')}
+				state={healthItems.automatic_taxes.state}
+				message={healthItems.automatic_taxes.message}
+			>
+				<FormSettingExplanation>
+					{healthItems.automatic_taxes.show_settings_link && (
+						<>
+							<ExternalLink
+								href="admin.php?page=wc-settings&tab=tax"
+							>
+								{translate('Go to the Tax settings')}
```

Comment:
Not a big one but can we display "Go to General settings" here instead in the `! wc_tax_enabled()` scenario?

Reply:
Right now I am just indicating wherever the settings link should be displayed. I guess I could pass the settings link text from the backend with a bit of refactoring 👍

Reply:
Or maybe an extra flag in the JSON object if that's simpler?